Docs/source/usage/parameters.rst

```diff
@@ -1753,9 +1753,12 @@ In-situ capabilities can be used by turning on Sensei or Ascent (provided they a
     divisor of ``blocking_factor``. If ``warpx.numprocs`` is used instead, the total number of cells in a given
     dimension must be a multiple of the ``coarsening_ratio`` multiplied by ``numprocs`` in that dimension.
 
-* ``<diag_name>.file_prefix`` (`string`) optional (default `diags/plotfiles/plt`)
+* ``<diag_name>.file_prefix`` (`string`) optional (default `diags/<diag_name>`)
     Root for output file names. Supports sub-directories.
 
+* ``<diag_name>.file_min_digits`` (`int`) optional (default `5`)
+    The minimum number of digits used for the iteration number appended to the diagnostic file names.
```

Comment:
Let's add the openPMD usage as well since it's fast to adopt?
```suggestion
* ``<diag_name>.file_min_digits`` (`int`) optional (default `5` for plotfiles and `6` for openPMD)
    The minimum number of digits used for the iteration number appended to the diagnostic file names.
```

As written on Slack, I fear the main practical limitation is that many plotfile tools hard-code 5 digits when reading plotfiles again #1087

Reply:
Did you mean to call this `file_num_digits`?

Reply:
`file_min_digits` is intended, since more digits will be added as needed if `log10(nsteps) > file_min_digits`

Reply:
Oh, didn't know. Ok